### DISCUSSION TITLE
Missing Features in „Figma-Version“ + Future Plans

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#76] [General]
Hallo!

I have very good feedback on the „Figma“-Version of this plugin, m especially to export local specs.

I dont realized that it also alows to export simpel prototypes. But it seems some Figma Features are missing here like inline-scroll or animated components. Do you plan to improve this plugin for Figma, do you have any similar plans?

regards!

**Comment by MaxBazarov** [comment]
Hello.
Yes, I’m planning to develop more features in Demo Station.
Currently I’m gathering a feedback to understand a future direction.

You mentioned inline-scroll or animated components. Ok. What else can be critical for you?

**Comment by Nevu** [comment]
Nice to hear that!

I will collect some more feedback and share it to you as soon as possible.
I think it would be Great to have all Features Figma-Prototypes have!

**Reply by Nevu** [reply]
I have some new issues here with Figma here:



colored frames aren't inspectable:
when export frames with backgground-color the color can't inspect – seems to be an image then

wrong 1px border:
if inspect a colored red, it is shown 1px when there is no border set in file (compare screenshot from within Figma and test-file)
[test.zip](https://github.com/ingrammicro/puzzle-publisher/files/8523223/test.zip)

**Comment by MaxBazarov** [comment]
Thanks for the report.
Just fixed both issues.

**Comment by Nevu** [comment]
Hallo again!

I tried the last version in Figma again.
It seems, there a still problems with inspect mode and groups/ frames/ stacks. If used, you can't „reach“ a deeper Element f.e. when creating a text-form!

Do you need an example file?

*BTW this plugin would be very precious for our Devs if you could fix some issues 👍

Regards

**Comment by MaxBazarov** [comment]
Yes. Example would be great to have.

**Reply by Nevu** [reply]
Heeres the [Figma-Link ](https://www.figma.com/file/XJWQkGNsrJ2mm7dcwIjlW8/puzzle-figma-test?node-id=0%3A1). Files are attached. There are two tries of ungrouping.

[puzzle-figma-test1.zip](https://github.com/ingrammicro/puzzle-publisher/files/8715406/puzzle-figma-test1.zip)
.

**Comment by MaxBazarov** [comment]
Hi.
Probably you don’t know, but you can click several times to iterate layers in a stack. 
Does it work for you?

**Reply by Nevu** [reply]
Oh ... nice 😅!
I will share this „trick“!

**Comment by MaxBazarov** [comment]
Let’s go to Demo Station page for future discussions.
https://github.com/MaxBazarov/demo-station-plugin/discussions
